lib/commands/find.js

```diff
@@ -36,7 +40,7 @@ helpers.findElOrEls = async function (strategy, selector, mult, context = '') {
     }
 
     // we want to return false if we want to potentially try again
-    if (mult) {
+    if (params.multiple) {
```

Comment:
This is fine, but also there is no need for the change (since `mult` exists in this scope now).